core/tests/webdriverio_utils/waitFor.js

```diff
@@ -194,7 +194,7 @@ var urlRedirection = async function(url) {
 var numberOfElementsToBe = async function(
     elementSelector, elementName, number, childSelector = null) {
   await browser.waitUntil(async function() {
-    var element = childSelector ? await $(elementSelector).$$(
+    var element = childSelector ? await elementSelector.$$(
       childSelector) : await $$(elementSelector);
     return element.length === number;
```

Comment:
I just noticed that there is no error message for this. Can you please add one?

Reply:
Please take a look at line no 203, we have error message present for this.